pkg/scheduler/cache/cache.go

```diff
@@ -152,14 +152,14 @@ func (db *defaultBinder) Bind(kubeClient *kubernetes.Clientset, tasks []*schedul
 	}
 
 	if len(errTasks) > 0 {
-		return fmt.Errorf("failed to bind pods"), errTasks
+		return errTasks, fmt.Errorf("failed to bind pods")
 	}
 
 	return nil, nil
 }
 
-func NewBinder() *defaultBinder {
-	return &defaultBinder{}
+func NewBinder() *DefaultBinder {
```

Comment:
This changes the scope of "DefaultBinder".

Reply:
> This changes the scope of "DefaultBinder".

Same as above.

Reply:
why change the scope ?